<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parachutes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <family>Fixedsys</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPause"/>
   <addaction name="actionStart"/>
   <addaction name="actionTimer"/>
   <addaction name="actionScore"/>
   <addaction name="actionOldScore"/>
   <addaction name="actionLevel"/>
  </widget>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/Pause.png</normaloff>
     <normalon>:/images/Pause.png</normalon>
     <activeon>:/images/start.png</activeon>
     <selectedon>:/images/Pause.png</selectedon>:/images/Pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/start.png</normaloff>:/images/start.png</iconset>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </action>
  <action name="actionTimer">
   <property name="text">
    <string>Timer:</string>
   </property>
   <property name="font">
    <font>
     <family>AR PL UMing TW MBE</family>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
  </action>
  <action name="actionScore">
   <property name="text">
    <string>Score: 0</string>
   </property>
   <property name="toolTip">
    <string>Score</string>
   </property>
   <property name="font">
    <font>
     <family>AR PL UMing CN</family>
    </font>
   </property>
  </action>
  <action name="actionOldScore">
   <property name="text">
    <string>oldScore</string>
   </property>
   <property name="font">
    <font>
     <family>AR PL UMing TW</family>
     <pointsize>15</pointsize>
    </font>
   </property>
  </action>
  <action name="actionLevel">
   <property name="text">
    <string>level</string>
   </property>
   <property name="font">
    <font>
     <family>Fixedsys</family>
     <pointsize>16</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
